### PR TITLE
fix(frontend): change global colors

### PR DESCRIPTION
**Pull request by denisquesada**
## Description

 A global style change that increase the contrast ratio to 4.8:1.
Change `#C05600` to `#BC4D01` wherever the hex code occurs on the app within CSS.

## Testing

Describe how you tested your changes and/or give instructions to the reviewers on how they can verify them.
Go to https://d2zdyv8kmq5ffn.cloudfront.net/admin/settings/brandingandstyling/editcolors
Verify that `#C05600` has been replaced by  `#BC4D01` in the **Data visualization second color (optional)** dropdown.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/components/PublishDashboardModal.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ function PublishDashboardModal(props: PathParams) {
                 }
             } catch (err) {
```
Comment (mabreuortega) [inline frontend/src/components/PublishDashboardModal.tsx:96]: err: any should do the same

File: frontend/src/components/Combobox.css [hunk 2]
```diff
@@ -14,9 +14,9 @@
   content: "pla";
 }
 
-.usa-combo-box__list-option[data-value="#c05600"]::after {
-  background-color: #c05600;
-  color: #c05600;
+.usa-combo-box__list-option[data-value="#BC4D01"]::after {
+  background-color: #BC4D01;
+  color: #BC4D01;
```
Comment (mabreuortega) [inline frontend/src/components/Combobox.css:19]: This should be using color from the theme itself, it is good for now but in the future this needs to be converted to SASS and use color("color_name") from the theme palette

File: frontend/src/services/DatasetParsingService.ts [hunk 3]
```diff
@@ -39,7 +39,7 @@ function getFilteredJson(json: Array<any>, hiddenColumns: Set<string>): Array<an
             obj[key] = row[key];
             return obj;
         }, {});
-        if (filteredRow !== {}) {
+        if (Object.keys(filteredRow).length) {
```
Comment (mabreuortega) [inline frontend/src/services/DatasetParsingService.ts:42]: nitpick: Object.keys(filteredRow).length > 0

**Review by mabreuortega — APPROVED** [review]
LGTM, suggestion added

File: frontend/src/components/PublishDashboardModal.tsx (hunk 1 above)
Comment (denisquesada) [reply frontend/src/components/PublishDashboardModal.tsx:96]: When adding `catch(err:any)` type scripts throws the error as follows:
```
TS1196: Catch clause variable cannot have a type annotation.

**Pull request merged**